examples/visual_gen/serve/async_video_gen.py

```
@@ -28,7 +28,7 @@ def test_async_video_generation(
     base_url: str = "http://localhost:8000/v1",
     model: str = "wan",
     prompt: str = "A video of a cool cat on a motorcycle in the night",
-    input_reference: str = None,
+    image_reference: str = None,
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)